gradio-blocks/README.md

```diff
@@ -0,0 +1,100 @@
+* Change a component's properties (for example, the choices in a Dropdown) or its visibility based on user input.
+
+To learn more about Blocks, see the [official guide](https://www.gradio.app/introduction_to_blocks/).
+
+
+**How Do Gradio and Hugging Face work together?**
+
+Hugging Face Spaces is a free hosting option for Gradio demos. Spaces comes with 3 SDK options: Gradio, Streamlit and Static HTML demos. Spaces can be public or private and the workflow is similar to github repos. There are over 2000+ Gradio spaces currently on Hugging Face. Learn more about spaces [here](https://huggingface.co/docs/hub/spaces).
+
+**Event Plan**
+
+main components of the event consist of:
+
+- Learning about Gradio and the new Blocks Feature
+- How Gradio and Hugging Face Spaces work together
+- Understanding the Hugging Face ecosystem
+- Share Demos in a Hugging Face organization for the Blocks event
+
+
+**Example spaces using Blocks**
+- [https://huggingface.co/spaces/akhaliq/ArcaneGAN-blocks](https://huggingface.co/spaces/akhaliq/ArcaneGAN-blocks)
+- [https://huggingface.co/spaces/merve/gr-blocks](https://huggingface.co/spaces/merve/gr-blocks)
+- [https://huggingface.co/spaces/osanseviero/tortoisse-tts](https://huggingface.co/spaces/osanseviero/tortoisse-tts)
+- [https://huggingface.co/spaces/akhaliq/CaptchaCracker](https://huggingface.co/spaces/akhaliq/CaptchaCracker)
+
+
+To participate in the event
+
+- Join the organization for Blocks event
+    - [https://huggingface.co/Gradio-Blocks](https://huggingface.co/Gradio-Blocks)
+- Join the discord
+    - [discord](https://discord.com/invite/feTf9x3ZSB)
+
+
+Participants will have a month to build and share Gradio demos using the Blocks feature. We will share a list of ideas of spaces that can be created using blocks or participants are free to try out their own ideas. At the end of the event, spaces will be evaluated for creativity and prizes will be given 
+
+
+potential ideas for creating spaces:
+
+
+- papers from https://paperswithcode.com/
+- themed spaces: Vision, NLP, Audio based spaces
+- Models from huggingface model hub
+- Models from other model hubs
+    - Tensorflow Hub: see example Gradio demos at https://huggingface.co/tensorflow
+    - Pytorch Hub: see example Gradio demos at https://huggingface.co/pytorch
+    - ONNX model Hub: see example Gradio demos at https://huggingface.co/onnx
+    - PaddlePaddle Model Hub: see example Gradio demos at https://huggingface.co/PaddlePaddle
+- participant ideas
+
+
+**Prizes (tentative list)**
+
+- HF PRO 1 month or 1 year!
+- 1st place 12 months, 2nd 6m, 3rd 3m, 4th 1m HF PRO.
+- Legendary prize, Lifetime HF PRO!!!
+- HF PRO badge on HF
+- Gradio badge on HF
+- Special event badge on HF
+- A lot more swag: t-shirts, hoodies, mugs
+- Embed the demo or highlight it
+- Time with someone on the Gradio team (unique ?)
```

Comment:
I'd rather call it something like office-hour or 1:1 meeting.

Reply:
Sounds good, will update to Gradio team office-hour

Reply:
updated